worlds/pokemon_rb/items.py

```diff
@@ -212,7 +212,13 @@ def __init__(self, item_id, classification, groups):
 )
 
 
-item_groups = {}
+item_groups = {
+    "HM01": {"HM01 Cut"},
+    "HM02": {"HM02 Fly"},
+    "HM03": {"HM03 Surf"},
+    "HM04": {"HM04 Strength"},
+    "HM05": {"HM05 Flash"},
+}
```

Comment:
Each item has a list of groups they're in. I would prefer we add them there instead of having two different ways of getting item name groups added.

Reply:
Sure I can do it that way.